src/index.html

```diff
@@ -26,10 +26,9 @@
           id="password"
          >
       </label>
-
       <button type="button" id="submit">Login</button>
     </form>
-
+    <div id="div"></div>
```

Comment:
Think you don't need to create it in HTML file, you should create that element directly by JS